spec/preservation/update_moab_spec.rb

```diff
@@ -0,0 +1,45 @@
+describe Preservation::UpdateMoab do
+  let(:pres_update_moab) { described_class.new }
+  let(:full_druid) { 'druid:bj102hs9687' }
+  let(:mock_path) { instance_double(Pathname) }
+  let(:mock_so) { instance_double(Moab::StorageObject, object_pathname: mock_path) }
+
+  context '#perform' do
+    it 'calls #update_moab' do
+      expect(LyberCore::Log).to receive(:debug).with("update-moab druid:bj102hs9687 starting")
+      expect(Moab::StorageServices).to receive(:find_storage_object).with(full_druid, true).and_return(mock_so)
+      expect(mock_path).to receive(:mkpath)
+      expect(pres_update_moab).to receive(:update_moab).with(mock_so)
+      pres_update_moab.perform(full_druid)
+    end
+  end
+
+  context '#update_moab' do
+    let(:mock_new_version) { instance_double(Moab::StorageObjectVersion) }
+    let(:verification_result) { instance_double(Moab::VerificationResult) }
+
+    before do
+      allow(Moab::StorageServices).to receive(:find_storage_object).and_return(mock_so)
+      allow(mock_path).to receive(:mkpath)
+    end
+
+    it 'calls verify_version_storage on Moab::StorageObjectVersion' do
```

Comment:
since it's crucial that it also calls `ingest_bag`, this should be in the test description, too.  :-)

It might be better if you can combine these tests with the #perform test (and thus, not test private methods) -- but I wouldn't spend more than an hour on it.  this is testing the right stuff.